### PR TITLE
Remove special forks annotations

### PR DESCRIPTION
**Pull request by hweawer**

### CODE REVIEW

File: .github/workflows/integration-tests.yml [hunk 1]
```diff
@@ -29,34 +29,16 @@ jobs:
...
-          ANVIL_PATH: ""
-
-      - name: Integration tests with pytest, mainnet fork
+      - name: Integration tests with pytest
         if: success() || failure()
         run: |
           poetry run pytest tests -m integration
         env:
           WEB3_RPC_ENDPOINTS: ${{ secrets.WEB3_RPC_ENDPOINT }}
+          TESTNET_WEB3_RPC_ENDPOINTS: 'https://ethereum-holesky-rpc.publicnode.com'
```
Comment (F4ever) [inline .github/workflows/integration-tests.yml:38]: Add this var to readme (test section)

File: .github/workflows/integration-tests.yml [hunk 2]
```diff
@@ -29,34 +29,16 @@ jobs:
...
           DEPOSIT_CONTRACT: "0x00000000219ab540356cBB839Cbe05303d7705Fa"
           LIDO_LOCATOR: "0xC1d0b3DE6792Bf6b4b37EccdcC24e45978Cfd2Eb"
+          ONCHAIN_TRANSPORT_RPC_ENDPOINTS: "https://gnosis-chiado-rpc.publicnode.com"
```
Comment (F4ever) [inline .github/workflows/integration-tests.yml:41]: and this one

Comment (hweawer) [reply .github/workflows/integration-tests.yml:41]: done

File: .github/workflows/integration-tests.yml (hunk 1 above)
Comment (hweawer) [reply .github/workflows/integration-tests.yml:38]: done

File: README.md [hunk 3]
```diff
@@ -119,6 +120,8 @@ poetry run pytest tests -m unit
 
 #### Run integration tests.
 
+TESTNET_WEB3_RPC_ENDPOINTS - set this variable for the Ethereum EL testnet RPC
```
Comment (F4ever) [inline README.md:123]: This should be Holesky only

Comment (hweawer) [reply README.md]: Done

**Review by F4ever — APPROVED** [review]

**Pull request merged**